server/pulp/server/controllers/repository.py

```diff
@@ -1157,13 +1143,36 @@ def check_publish(repo_obj, dist_id, dist_inst, transfer_repo, conduit, call_con
             repo_id=repo_obj.repo_id,
             distributor_id=dist_id).update(set__last_override_config=config_override)
 
+    skip_for_predistributor = False
     # Check if a predistributor is configured and the predistributor has not published since the
     # last publish.
-    skip_for_predistributor = (predistributor_id and (published_after_predistributor or
```

Comment:
This line referenced the variables ```published_after_predistributor``` and ```predistributor_last_published```, both of which were defined behind the conditional beginning on L1146.